lib/cretonne/src/ir/instructions.rs

```diff
@@ -333,6 +333,56 @@ impl InstructionData {
         }
     }
 
+    /// Get a mutable reference to the value list of jump arguments of this branch.
+    ///
+    /// Multi-destination branches like `br_table` return `None`.
+    /// TODO: deal with multi-branch instructions
+    pub fn branch_arguments_mut(&mut self, dest: Ebb) -> Option<&mut ValueList> {
```

Comment:
Ok, managed a solution using `take_value_list()` and `analyze_branch()` because I only need to append an value to the argument value list, No need to generate `value_list_mut()`.